src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,194 @@
+package subway;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.jdbc.Sql;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@DisplayName("노선관련 기능")
+@Sql(scripts = {"StationInsert.sql", "LineInsert.sql"})
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class LineAcceptanceTest {
+
+    /**
+     * Given: 새로운 지하철 노선 정보를 입력하고,
+     * When: 관리자가 노선을 생성하면,
+     * Then: 해당 노선이 생성되고 노선 목록에 포함된다.
+     */
+    @DisplayName("지하철 노선을 생성한다.")
+    @DirtiesContext
+    @Test
+    void 지하철_노선_생성_테스트() {
+        //given
+        Map<String, String> params = new HashMap<>();
+        String lineName = "GTX-A";
+        String color = "burgundy";
+        String upStationId = "1";
+        String downStationId = "2";
+        String distance = "20";
+
+        params.put("name", lineName);
+        params.put("color", color);
+        params.put("upStationId", upStationId);
+        params.put("downStationId", downStationId);
+        params.put("distance", distance);
+
+        //when
+        ExtractableResponse<Response> response =
+                RestAssured.given().log().all()
+                        .body(params)
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .when().post("/lines")
+                        .then().log().all()
+                        .extract();
+
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        //then
+        List<String> lineList = getAPIResponse("/lines").jsonPath().getList("name", String.class);
```

Comment:
- 모든 지하철 노선의 '이름'을 갖는 List이니, 변수의 이름을 `lineNames` 정도로 지어보면 어떨까요?
- `getLines()`를 활용해보면 어떨까요?
```suggestion
        List<String> lineNames = getLines();
```